### ISSUE TITLE
[Bot Channels Registration] Cannot find app secret when creating SDK bot through Azure Portal

### ISSUE THREAD

**Issue by srozga** [Microsoft/BotBuilder#3938]
Congratulations on the GA release of the bot service and LUIS!

I'm trying to create a new bot registration, and obviously, dev.botframework.com doesn't let me anymore. So I follow the process of creating a Bot Channels Registration in the Azure Portal. Everything goes well. I realize I don't have my app id/secret. I find the app id in settings. Great. Still no secret. So I click the manage link and find it hidden there. 

No problem, I figure let's create a new password but I get this error since yesterday (Friday 12/15) afternoon:
![image](https://user-images.githubusercontent.com/1940385/34070185-06710970-e22f-11e7-947c-a15d9e90067c.png)

So, basically can't create bot registrations anymore. 

Can you support?
Thanks
-s

**Comment by EricDahlvang** [comment]
Thanks for reporting this to us. This underlying issue has been identified and is being addressed by the team. We apologize for the inconvenience.

**Comment by nwhitmont** [comment]
@srozga check out
https://docs.microsoft.com/en-us/bot-framework/bot-service-quickstart-registration#bot-channels-registration-password

**Comment by srozga** [comment]
Thanks @nwhitmont . Great doc. I'll keep it handy.

**Comment by srozga** [comment]
@nwhitmont @EricDahlvang Happy New Year! I'm still seeing the same issue. Graph Exception when I try to create a new password. This issue should be reopened.

**Comment by EricDahlvang** [comment]
@srozga 

The fix for this is currently being rolled out.  I've verified that I'm able to create a new password for a bot in West US.  Which data center is your bot in?

**Comment by srozga** [comment]
@EricDahlvang: CentralUS

**Comment by EricDahlvang** [comment]
@srozga Would you please try again now.  I've checked our internal system, and all data centers have been updated as of 2:20pm PST.

**Comment by srozga** [comment]
@EricDahlvang confirmed! thanks a bunch!
